README.md

```diff
@@ -526,7 +549,7 @@ $ ls -al samples/library/build//outputs/aar/library-debug.aar
 
 To test in a real project, use the local Maven repository in your `build.gradle`, like:
 
-```
+```gradle
 buildscript {
     repositories {
         maven {
```

Comment:
Bump the `samples/maven-repo` below to `build/local-repo` while you're here, please.

Reply:
And the version number.